pyseq.py

```diff
@@ -66,9 +66,12 @@
 # regex for matching format directives
 format_re = re.compile(r'%(?P<pad>\d+)?(?P<var>\w+)')
 
+# character to join explicit frame ranges on
+range_join = os.environ.get("PYSEQ_SEP", " ")
```

Comment:
I'm not sure how common it is but I brought it up because I've always used commas instead of spaces for broken frame ranges.  I guess that is pretty generic name, maybe PYSEQ_FRAME_SEP?